var/spack/repos/builtin/packages/ibmisc/package.py

```diff
@@ -29,11 +29,17 @@ class Ibmisc(CMakePackage):
     """Misc. reusable utilities used by IceBin."""
 
     homepage = "https://github.com/citibeth/ibmisc"
-    url      = "https://github.com/citibeth/ibmisc/archive/v0.1.0.tar.gz"
+    url      = "https://codeload.github.com/citibeth/ibmisc/tar.gz/v0.1.0.tar.gz"
```

Comment:
Can you match the version in this URL to the version listed below? We have URL parsing tests that look for this.